src/.env.example

```diff
@@ -13,6 +13,10 @@ DB_DATABASE=laravel
 DB_USERNAME=laravel
 DB_PASSWORD=passPRO
 
+PASSPORT_LOGIN_ENDPOINT=http://localhost/oauth/token
```

Comment:
localhost should be replaced by dev_nginx to work

Reply:
Done